pkg/auth/user/RoleGroupService.go

```diff
@@ -732,8 +734,12 @@ func (impl RoleGroupServiceImpl) DeleteRoleGroup(bean *bean.RoleGroup) (bool, er
 	if err != nil {
 		impl.logger.Errorw("error in getting all roles for groups", "err", err)
 	}
-	for _, roleGroupRoleMapping := range allRoleGroupRoleMappings {
-		err = impl.roleGroupRepository.DeleteRoleGroupRoleMappingByRoleId(roleGroupRoleMapping.RoleId, tx)
+	roleGroupRoleMappingIds := make([]int, 0, len(allRoleGroupRoleMappings))
+	for _, roleMapping := range allRoleGroupRoleMappings {
+		roleGroupRoleMappingIds = append(roleGroupRoleMappingIds, roleMapping.Id)
+	}
```

Comment:
can directly get only ids from db, since allRoleGroupRoleMappings is not being used anywhere else

Reply:
DONE